src/tools/miri/src/shims/files.rs

```diff
@@ -2,7 +2,7 @@ use std::any::Any;
 use std::collections::BTreeMap;
 use std::fs::{File, Metadata};
 use std::io::{ErrorKind, IsTerminal, Seek, SeekFrom, Write};
-use std::marker::CoercePointee;
+use std::ops::CoercePointee;
```

Comment:
I fixed the path used by `FileDescriptionRef` so I don't understand why `CoercePointee` broke for `FileDescriptionRef`. :(

Reply:
I think you'll need `#[cfg(bootstrap)] std::marker::CoercePointee` and `#[cfg(not(bootstrap))] std::ops::CoercePointee` so that miri can build stage 1 with an older std and stage 2 with a newer std. Check everything using `./x check --stage 2`.